src/utils/todo/task-filing/button-loader.ts

```diff
@@ -1,10 +1,15 @@
 import { FilingTypes } from '@bcrs-shared-components/enums'
-import { filingTypeToName } from './helper'
+import { filingTypeToName, isStaffTodo } from './helper'
 import * as actionFunctions from '~/utils/todo/action-functions'
 
 /** Add actionButton to the todo item */
 // https://docs.google.com/spreadsheets/d/1rJY3zsrdHS2qii5xb7hq1gt-D55NsakJtdu9ld9d80U/edit?gid=792248919#gid=792248919
 export const addActionButton = (todoItem: TodoItemI): void => {
+  // non-staff see no buttons for staff filings (cont out, conversion, correction, restoration)
+  if (!useBcrosAccount().isStaffAccount && isStaffTodo(todoItem)) {
```

Comment:
below on 99 you import isStaffAccount. and in most other places you have used the import instead so you should raise the import and use it here